.github/workflows/ci.yaml

```diff
@@ -70,6 +70,12 @@ jobs:
       - name: Test
         run: cargo test --features sysroot-abi -p proc-macro-srv -p proc-macro-srv-cli -p proc-macro-api -- --quiet
 
+      - run: |
+          if cargo tree -p proc-macro-srv-cli | grep -q 'salsa'; then
+            echo "The proc-macro server may not depend on salsa"
+            exit 1
+          fi
```

Comment:
can this be done more concisely?

Reply:
`! (cargo tree -p proc-macro-srv-cli | grep -q salsa)` maybe?